src/pages/non-pass-finalisation/cat-be/non-pass-finalisation.cat-be.page.html

```diff
@@ -0,0 +1,44 @@
+<ion-header>
+  <practice-mode-banner *ngIf="isPracticeMode"></practice-mode-banner>
```

Comment:
can be removed when base page is updated

Reply:
Done